controller.py

```diff
@@ -272,14 +272,15 @@ def library_details(self, uuid, library=None):
         hyperlink = Library.get_hyperlink(library, Hyperlink.INTEGRATION_CONTACT_REL)
         contact_email = self._contact_email(hyperlink)
         validated_at = self._validated_at(hyperlink)
-
+        pls_id = ConfigurationSetting.for_library("pls_id", library)
```

Comment:
"pls_id" is a cryptic string that's used a lot; it would be good to keep it as a constant in `Library` with a comment explaining what it's for.